### ISSUE TITLE
resource.distribution.contact.present

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-checks#54]
## Description

Check if a distribution contact exists, such as an email, URL or physical address. This would be used to obtain data entities that are not available via a provided URL.

## Priority

- FAIR: Required

## Issues
- EML does not provide this

## Procedure

- check if a distribution contact is present

**Comment by tedhabermann** [comment]
The target for this check exists in the DataCite Dialect: 
data[\*].attributes.contributors[?(@.contributorType='Distributor')].name, data[\*].attributes.contributors[?(@.contributorType='Distributor')].givenName,
data[*].attributes.contributors[?(@.contributorType='Distributor')].familyName

**Comment by JEDamerow** [comment]
Is this check different from #122 ?

**Comment by jeanetteclark** [comment]
I think the EML path for this check should also include `/eml/*/contact/`

@mbjones concurred on slack

**Comment by gothub** [comment]
@jeanetteclark the xpath has been added in commit dbbfd83cdd8934cb3a034f4dc07c813ccada0ac2
